openedx/adg/lms/applications/management/commands/update_is_prerequisite_courses_passed.py

```diff
@@ -25,118 +27,51 @@ def handle(self, *args, **kwargs):  # pylint: disable=unused-argument
 
-    def get_minimal_users_to_be_checked_for_update(self, prereq_course_groups):
-        """
-        Filter minimal set of users who should be checked for the update.
-        Args:
-            prereq_course_groups: List of course groups
-        Returns:
-            A list of user ids who should be checked for the update
-        """
-        users_to_be_checked_for_update = self.get_users_with_pre_reqs_not_marked_as_passed()
-
-        for prereq_course_group in prereq_course_groups:
-            users_to_be_checked_for_update = self.get_users_with_active_course_enrollments(
-                users_to_be_checked_for_update, prereq_course_group
-            )
-
-        return users_to_be_checked_for_update
-
     def get_users_with_pre_reqs_not_marked_as_passed(self):
         """
-        Get users who do not have associated application hub and if they have, then they have not passed all prereq
-        courses
+        Get users who have completed their written application but not passed the program prereq courses
         Returns:
-            A querySet for user ids whose pre req courses are not yet marked as passed
+            A querySet for user ids whose program pre req courses are not yet marked as passed
         """
         return User.objects.filter(
-            Q(application_hub__isnull=True) | Q(application_hub__is_prerequisite_courses_passed=False),
+            application_hub__is_written_application_completed=True,
+            application_hub__is_prerequisite_courses_passed=False
         ).values_list('id', flat=True)
 
-    def get_users_with_active_course_enrollments(self, users_to_be_checked_for_update, prereq_course_group):
-        """
-        Filter users from provided user ids. Filtered users that enrolled in one or more courses from course group
-        and has not unenrolled course
-        Args:
-            users_to_be_checked_for_update: List of user ids
-            prereq_course_group: PrerequisiteCourseGroup model object
-        Returns:
-            A querySet for users who should be checked for the update
-        """
-        course_keys = prereq_course_group.open_multilingual_course_keys()
-        return User.objects.filter(
-            id__in=users_to_be_checked_for_update,
-            courseenrollment__course__in=course_keys,
-            courseenrollment__is_active=True,
-        ).values_list('id', flat=True).distinct()
-
-    def check_users_for_application_update(self, users_to_be_checked_for_update, prereq_course_groups):
+    def check_users_for_application_update(self, users_to_be_checked_for_update):
         """
-        Check check passed prereq courses for all filtered users
+        Check user has passed prereq courses and update application hub flag
         Args:
             users_to_be_checked_for_update: List of user ids
-            prereq_course_groups: List of course groups
         Returns:
             None
         """
```

Comment:
Lets follow the convention and add spaces between function definition, Args and Returns